asteroid/dsp/overlap_add.py

```diff
@@ -154,13 +154,12 @@ def _reorder_sources(
     previous = previous.reshape(-1, n_src, frames)
 
     overlap_f = window_size - hop_size
-    pw_losses = PITLossWrapper.get_pw_losses(
-        lambda x, y: torch.sum((x.unsqueeze(1) * y.unsqueeze(2))),
```

Comment:
I realize now that this might be weird. 
It constructs a matrix of pointwise products of target vs. estimates and makes the total sum, even with the batch.. I think it is wrong, what was the intention @popcornell ?